### DISCUSSION TITLE
Missing values imputation

### DISCUSSION THREAD

**Discussion by asafbenj** [AdaptiveMotorControlLab/CEBRA#138] [Q&A]
Hi, I noticed CEBRA doesn't accept missing values. What are good strategies for replacing them? Any CEBRA-specific considerations or should we approach this as a general missing values imputation task?

**Comment by stes** [comment]
Hi @asafbenj , depends where your missing data is -- in the neural data, the behaviour/stimuli, or both?

**Reply by asafbenj** [reply]
In my input data, which is actually behavioral data, unlike the standard CEBRA use case example. I'm trying to learn an embedding of behavior that's useful for predicting neural activity, rather than vice versa.

**Reply by MMathisLab** [reply]
Check out the provided DeepLabCut (behavior only) loader; there we handle low confidence keypoints.
